adagrams/game.py

```diff
@@ -1,85 +1,113 @@
 from random import randint
-from adagrams.asset import LETTER_POOL
-
+# from adagrams.asset import LETTER_POOL
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+HAND_SIZE = 10 # define constant to tacke magic number
 
 def draw_letters():
-    #create a copy of pool for tracking
+    # create a copy for tracking pool
     tracking_pool = []
-    for key, value in LETTER_POOL.items():
-        tracking_pool.append({key: value})
+    # use list instead of dict
+    for letter, count in LETTER_POOL.items():
+        # create flat list using letter only for probability ["A", "A", "A", "B", "B"]
+        tracking_pool += [letter] * count 
+        # tracking_pool.append([key, value]) this would create inside list
+
+    print("tracking_pool:", tracking_pool)
     
-    letters = [] #create like ["A", "K", "S", "O", "R", "E", "T", "A", "I", "N"]
+    letters = [] # this will be returned
     
-    # counter = 1
-    while len(letters) < 10: # when 9 len, the loop run + if len becomes 10, loop stops
-        
-        random_index = randint(0, len(tracking_pool) -1) # 8
-        dict_item = tracking_pool[random_index] # {'I': 9 }
-        
-        #in {'I': 9 }
-        for key, value in dict_item.items(): 
-            # if value > 0, append the key and value-1
-            if value > 0:
-                letters.append(key)
-                value -= 1
-                tracking_pool[random_index][key] = value #update the value in the tracking_pool
-            #if value == 0, loop continue until next one is picked
-        
-        # counter += 1
+    while len(letters) < HAND_SIZE:
+        random_index = randint(0, len(tracking_pool) - 1) # 8
+        # once you append letter by index, remove it from pool
+        letters.append(tracking_pool[random_index])
+        tracking_pool.pop(random_index)
     
-    # print("tracking_pool", tracking_pool)
     print("letters:", letters)
     return letters
     
-# draw_letters()
+draw_letters()
```

Comment:
Watch out for stray function calls in the module. This will run when the module gets imported.